python/llm/test/benchmark/csv_to_html.py

```diff
@@ -106,11 +146,17 @@ def main():
         latest_csv.insert(loc=5,column='last2',value=last2)
         latest_csv.insert(loc=6,column='diff2(%)',value=diff2)
 
-        diffs_within_normal_range = is_diffs_within_normal_range(diff1, diff2, highlight_threshold)
+        latest_csv.insert(loc=7,column='best 1',value=best_last1)
+        latest_csv.insert(loc=8,column='best diff1(%)',value=best_diff1)
+        latest_csv.insert(loc=9,column='best 2',value=best_last2)
+        latest_csv.insert(loc=10,column='best diff2(%)',value=best_diff2)
+
+        diffs_within_normal_range = is_diffs_within_normal_range(diff1, diff2, best_diff1, best_diff2, highlight_threshold)
```

Comment:
There is no need to add best_diff1 and best_diff2 in this function argument :)

Reply:
Thanks for reminding！